crypto/nrf_cc310_mbedcrypto/include/mbedtls/ecp_alt.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2001-2019, Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause OR Arm’s non-OSI source license
+ */
+
```

Comment:
Missing header guard.

Reply:
Missing in the original file as well, will fix and report back to original project